src/Form/Control/UploadImage.php

```diff
@@ -58,11 +56,11 @@ public function setThumbnailSrc($src): void
     /**
      * Clear the thumbnail src.
      * You can also supply a default thumbnail src.
-     *
-     * @param string $defaultThumbnail
      */
-    public function clearThumbnail($defaultThumbnail = null): void
+    public function clearThumbnail(string $defaultThumbnail = null): void
     {
+        $defaultThumbnail ??= $this->defaultSrc;
```

Comment:
I would prefer here `if ($defaultThumbnail === null) {}` to stress this codepath is used if the arg is empty, not only more readable, but coverage can be also measured better.